install/kubernetes/helm/istio/values.yaml

```diff
@@ -18,8 +18,8 @@ global:
     image: proxyv2
     resources:
       requests:
-        cpu: 100m
-        memory: 128Mi
+        cpu: 10m
```

Comment:
does it will impact the performance of proxy if setting cpu as 10m?

Reply:
It shouldn't. Just notifies the scheduler that the proxy container needs at least 10m.
The 100m/128Mi were also arbitrarily set for the same purpose of enabling HPA to pods that contain the proxy. (in a nutshull: because all containers must have at least the requested cpu the proxy container used to block customers from having HPA on their injected pods).